Gini-iOS-SDK/GINIDocumentMetadata.h

```diff
@@ -0,0 +1,27 @@
+//
+//  GINIDocumentMetadata.h
+//  Gini-iOS-SDK
+//
+//  Created by Enrique del Pozo Gómez on 10/24/18.
+//
+
+@interface GINIDocumentMetadata: NSObject
+    @property (readonly) NSDictionary<NSString *, NSString *> *headers;
+
+/**
+ * The document metadata initializer.
+ *
+ * @param branchId              The branch id
```

Comment:
You should also mention BLZ and/or Bankleitzahl because that's what it's actually going to be.